content/docs/use-cases/data-registry/index.md

```diff
@@ -0,0 +1,46 @@
+# Data Registry
+
+One of the main uses of <abbr>DVC repositories</abbr> is the
+[versioning of data and model files](/doc/use-cases/data-and-model-files-versioning).
+DVC also enables cross-project [reusability](/doc/start/data-and-model-access)
+of these <abbr>data artifacts</abbr>. This means that your projects can depend
+on data from other repositories — like a **package management system for data
+science**.
+
+![](/img/data-registry.png) _Data management middleware_
+
+We can build a <abbr>DVC project</abbr> dedicated to versioning _datasets_ (or
+data features, [ML models](/doc/use-cases/model-registry), etc.). The repository
+contains the necessary metadata, as well as the entire change history. The data
+itself is stored in one or more [DVC remotes][remote storage]. This is what we
+call a **data registry** -- data management _middleware_ between ML projects and
+cloud storage. Advantages:
+
+- **Reusability**: Reproduce and organize _feature stores_ with a simple CLI
+  (`dvc get` and `dvc import` commands, similar to software package management
+  like `pip`).
+- **Persistence**: Separating metadata from storage on reliable platforms (Git,
+  cloud locations) improve the durability of your data.
+- **Storage optimization**: Centralize data shared by multiple projects in a
+  single location (distributed copies are possible too). This simplifies data
+  management and optimizes space requirements.
+- **Data as code**: Leverage Git workflows such as commits, branching, pull
+  requests, reviews, and even
+  [CI/CD for your data and models lifecycle](/doc/use-cases/ci-cd-for-machine-learning).
+  Think "Git for cloud storage".
+- **Security**: DVC-controlled [remote storage] (e.g. Amazon S3) can be
+  configured to limit data access. For example, you can setup read-only
+  endpoints (e.g. an HTTP server) to prevent data deletions or alterations.
+
+[remote storage]: /doc/command-reference/remote
+
+👩‍💻 Intrigued? Try our [registry tutorial] to learn how DVC looks and feels
+firsthand.
+
```

Comment:
Hmmm. Git thinks I created this new file but in fact it was just the first half of what I split. Nothing changed except this new link to its tutorial.